Library/Formula/mariadb.rb

```diff
@@ -80,6 +80,11 @@ def install
       args << "-DPLUGIN_TOKUDB=NO"
     end
 
+    # disable Mroonga on 10.1
```

Comment:
Why does it need to be disabled?

(The conditional can also be made into a single line: `args << "-DWITHOUT_MROONGA_STORAGE_ENGINE=1" if build.devel?`)